Community-Supported/adjust-vertex-order/README.md

```diff
@@ -0,0 +1,96 @@
+# adjust-vertex-order
+## __adjust_vertex_order__
+
+![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-53bd92.svg)
+
+__Current Version__: 1.0
+
+This sample demonstrates how you can adjust the vertex order of all polygons of an existing `.hyper` file. This is done by copying all of the tables and data into a new file while calling a SQL function (`geo_auto_vertex_order` or `geo_invertex_order`) on all columns of type `GEOGRAPHY`.
+
+For more information on Hyper's SQL functions that operate on `GEOGRAPHY` values, see [Geographic Functions](https://help.tableau.com/current/api/hyper_api/en-us/reference/sql/functions-geography.html).
+
+## Background
+
+For the `GEOGRAPHY` type, in case of polygons, the order of vertices in the polygon's rings defines what is inside and what is outside the polygon. 
+Tableau's convention is "interior-left", which means that as we progress through the vertices of the polygon in storage order, the interior of the polygon is to the left.
+In the "spheroidal" system, as used by Tableau, there is no such thing as "incorrect vertex ordering". Either ordering can be correct because in this system, reverse vertex ordering merely changes what space the polygon encloses.
+Hence, there is no way to automatically detect the intention of what space a polygon is intended to enclose.
+Depending on the data source used to import `GEOGRAPHY`, Tableau users may experience problems when their `.hyper` files contain `GEOGRAPHY` values with polygons whose vertices are in the reverse order of what is intended by the user.
+For example, depending on how the polygons are specified, Tableau may not automatically zoom the map to the data. Another example is with spatial joins which would also produce unexpected results.
+
+The image below depicts the problem with auto zoom for a collection of polygons, specified with an "interior-right" winding order, marking the boundaries of the European continent:
+
+![Europe interior-right](europe_interior_right.png)
+
+The image below depicts the desired case where the polygons are specified with an "interior-left" winding order:
+
+![Europe interior-left](europe_interior_left.png)
+
+## Script semantics
+
+This sample script connects to two `.hyper` files, an input file and an output file, and copies all the schemas, tables and data from the input file into the (newly created) output file. During the copy, either `geo_auto_vertex_order` or `geo_invert_vertex_order` is called on all columns of type `GEOGRAPHY`. The copying of data from source to target tables is done entirely inside Hyper, i.e. no data (apart from table definitions) is exchanged between the Hyper and Python processes.
```

Comment:
doesn't the script create the output file? Sounds like it would only connect to it?